<launch>

<!--    <include file="$(find lasr_vision_yolov8)/launch/service.launch">-->
<!--        <arg name="debug" value="true" />-->
<!--        <arg name="preload" value="['yolov8n.pt']" />-->
<!--    </include>-->

    <node pkg="qualification" type="object_position.py" name="object_position" output="screen"/>

    <node pkg="qualification" type="object_loc.py" name="object_loc" output="screen"/>

</launch>

